ddcz/models/used/creations.py

```diff
@@ -19,6 +19,8 @@
 from ..magic import MisencodedCharField, MisencodedTextField, MisencodedIntegerField
 from .users import UserProfile
 
+PUBLISHED = "a"
+
 APPROVAL_CHOICES = (
     ("a", "Schváleno"),
```

Comment:
I think `PUBLISHED` should be used here to denote it's what the constant is about. And since doing it, I'd maybe also add `n`, which should be...hmm, SUBMITTED? NOT_PUBLISHED?

Reply:
Added unpublished and approval choices are not dependent on those constants.